test/kafka-ssl/smoketest.td

```diff
@@ -146,31 +146,34 @@ exact:error publishing kafka schemas for sink: unable to publish value schema to
   ENVELOPE DEBEZIUM
 contains:self signed certificate in certificate chain
 
+> CREATE SECRET client_cert AS '${arg.client-cert}'
+> CREATE SECRET client_key AS '${arg.client-key}'
+> CREATE SECRET ca_cert AS '${arg.ca-cert}'
+
 # Ensure that connectors work with SSL basic_auth
 > CREATE CONNECTOR kafka_ssl
-  FOR KAFKA BROKER 'kafka:9092'
-  WITH (
-      security_protocol = 'SSL',
-      ssl_key_location = '/share/secrets/materialized.key',
-      ssl_certificate_location = '/share/secrets/materialized.crt',
-      ssl_ca_location = '/share/secrets/ca.crt',
-      ssl_key_password = 'mzmzmz'
```

Comment:
This `mzmzmz` is not being passed to the `CREATE CONNECTOR` statement, so I think the entire test only works because the key password it is being passed to Mz globally with the `SSL_KEY_PASSWORD` variable.

It is not even clear how one would reference the `mzmzmzm` in the `CREATE CONNECTOR` statement.

Reply:
Good catch, the necessary field exists on the struct and should get populated correctly I just did not actually specify it here. I will update this test case to specify it!